devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -185,6 +192,13 @@ def poll() {
 	zwave.switchMultilevelV1.switchMultilevelGet().format()
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+	refresh()
```

Comment:
Do you understand what this `(getDataValue("MSR")` does? It shouldn't affect the DTH from generating an event, right?